redis-server.py

```diff
@@ -1,23 +1,150 @@
 # redis-server.py
 
 import socket
+import argparse
+import threading
+import time
 
 from utils import server_response_decode
 
 HOST = "127.0.0.1"  # Standard loopback interface address (localhost)
-PORT = 65430  # Port to listen on (non-privileged ports are > 1023)
-MY_DICT = {}
+PORT = 6380  # Port to listen on (non-privileged ports are > 1023)
+DATA_DICT = {}  # Dictionary for storage of data on "Redis" server
+TIME_DICT = {}  # Dictionary for storing key and time (value) for SET time based options
+BUFFER_SIZE = 1024
+SEPARATOR = "\r\n"
+BULK_STR_START = "$"
+NIL_REPLY = "$-1\r\n"
+SYNTAX_ERROR_MSG = "-ERR syntax error\r\n"
+TIME_OPTIONS = set(["EX", "PX", "EXAT", "PXAT", "KEEPTTL"])
+SET_OPTIONS = set(["EX", "PX", "EXAT", "PXAT", "KEEPTTL", "XX", "NX", "GET"])
```

Comment:
reuse `TIME_OPTIONS` like `set(TIME_OPTIONS + ["XX", "NX", "GET"])`

Reply:
moved

Reply:
this was not updated

Reply:
fixed again